BrainPortal/app/views/data_providers/new.html.erb

```diff
@@ -56,7 +56,7 @@
     <span title="Type of data provider.">
 
       <p><%= f.label :type, "Type" %><br/>
-      <%= f.select :type, grouped_options_for_select(@typelist), { :prompt => "Select Provider Type" }, { :disabled => (params[:action].to_sym == :edit) } %><br/>
+      <%= f.select :type, grouped_options_for_select(@typelist), { :prompt => "Select Provider Type" } %><br/>
```

Comment:
Why that change?

Reply:
OK I think you're right. This must have been old code because that form cannot be used for editing anymore. Thanks.